packages/esm-patient-registration-app/src/patient-registration/patient-registration.test.tsx

```diff
@@ -177,260 +178,274 @@ const fillRequiredFields = async () => {
-        <Router>
-          <PatientRegistration isOffline={false} savePatientForm={jest.fn()} />
-        </Router>
-      </ResourcesContext.Provider>,
-    );
-    await waitFor(() => expect(screen.getByLabelText(/Demographics Section/)).not.toBeNull());
-    expect(screen.getByLabelText(/Contact Info Section/)).not.toBeNull();
-  });
+    it('has the expected sections', async () => {
+      render(
+        <ResourcesContext.Provider value={mockResourcesContextValue}>
+          <Router>
+            <PatientRegistration isOffline={false} savePatientForm={jest.fn()} />
+          </Router>
+        </ResourcesContext.Provider>,
+      );
+      await waitFor(() => expect(screen.getByLabelText(/Demographics Section/)).not.toBeNull());
+      expect(screen.getByLabelText(/Contact Info Section/)).not.toBeNull();
+    });
 
-  it('saves the patient without extra info', async () => {
-    const user = userEvent.setup();
-
-    render(
-      <ResourcesContext.Provider value={mockResourcesContextValue}>
-        <Router>
-          <PatientRegistration isOffline={false} savePatientForm={FormManager.savePatientFormOnline} />
-        </Router>
-      </ResourcesContext.Provider>,
-    );
-
-    await fillRequiredFields();
-    await user.click(await screen.findByText('Register Patient'));
-    await waitFor(() => {
-      expect(mockedSavePatient).toHaveBeenCalledWith(
-        expect.anything(),
-        expect.objectContaining({
-          identifiers: [], //TODO when the identifer story is finished: { identifier: '', identifierType: '05a29f94-c0ed-11e2-94be-8c13b969e334', location: '' }
-          // identifiers: [{ identifier: '', identifierType: '05a29f94-c0ed-11e2-94be-8c13b969e334', location: '' }],
-          person: {
-            addresses: [{}],
-            attributes: [],
-            birthdate: '1993-8-2',
-            birthdateEstimated: false,
-            gender: 'M',
-            names: [{ givenName: 'Paul', middleName: '', familyName: 'Gaihre', preferred: true, uuid: undefined }],
-            dead: false,
-            uuid: expect.anything(),
-          },
-          uuid: expect.anything(),
-        }),
-        undefined,
+    it('saves the patient without extra info', async () => {
+      const user = userEvent.setup();
+
+      render(
+        <ResourcesContext.Provider value={mockResourcesContextValue}>
+          <Router>
+            <PatientRegistration isOffline={false} savePatientForm={FormManager.savePatientFormOnline} />
+          </Router>
+        </ResourcesContext.Provider>,
       );
+
+      await fillRequiredFields();
+      await user.click(await screen.findByText('Register Patient'));
+      await waitFor(() => {
+        expect(mockedSavePatient).toHaveBeenCalledWith(
+          expect.objectContaining({
+            identifiers: [], //TODO when the identifer story is finished: { identifier: '', identifierType: '05a29f94-c0ed-11e2-94be-8c13b969e334', location: '' }
```

Comment:
Hi @vasharma05, Could you please elaborate on this TODO comment? (Haven't we implemented the other identifier types yet?)